src/components/PhotoGallery/PhotoGallery.tsx

```diff
@@ -1,66 +1,39 @@
-import axios from "axios";
-import { useCallback, useEffect, useState } from "react";
 import { PicturesContainer, PicturesHeader } from "./PhotoGallery.styles";
 import { LazyLoadImage } from "react-lazy-load-image-component";
-import Loading from "../Loading";
+// import Loading from "../Loading";
```

Comment:
If you remove Loading here, please delete the line, and move the loading logic to the index